addons/website_form/static/src/snippets/s_website_form/001.scss

```diff
@@ -0,0 +1,79 @@
+.editor_enable .s_website_form[data-vcss="001"] {
+    // Hidden field is only partially hidden in editor
+    .o_website_form_field_hidden {
+        display: block;
+        opacity: 0.5;
+    }
+}
+
+.s_website_form[data-vcss="001"] {
+    // Radio buttons and checkboxes flex layout
+    .o_website_form_flex {
+        display: flex;
+        flex-wrap: wrap;
+
+        &.o_website_form_flex_fw > .o_website_form_flex_item {
+            flex-basis: 100%;
+        }
+        &:not(.o_website_form_flex_fw) > .o_website_form_flex_item {
+            // col-lg-4
+            flex-basis: 33%;
+
+            // col-md-6
+            @include media-breakpoint-down(md) {
+                flex-basis: 50%;
+            }
+
+            // col-12
+            @include media-breakpoint-down(sm) {
+                flex-basis: 100%;
+            }
+        }
+    }
+
+    .o_website_form_field_hidden {
+        display: none;
+    }
+
+    // Required fields have a star which is not part of the field label
+    .o_we_required_star {
+        .s_website_form_field {
+            &.o_website_form_required, &.o_website_form_required_custom {
+                .o_we_form_label:after {
+                    content: ' *';
+                }
+            }
+        }
+    }
+    // Optional fields have an indication which is not part of the field label
+    .o_we_required_opt {
+        .s_website_form_field {
+            &:not(.o_website_form_required):not(.o_website_form_required_custom) {
+                .o_we_form_label:after {
+                    content: ' - Opt.';
```

Comment:
Is it possible to do that in SCSS or should I do it in JS ?

Reply:
Could be done with a hack using content: attr(...); reading a translated attribute but since the DOM comes from a translated user content, I doubt you will have a translated attribute available. So still need JS to add that translated attribute on the element... but in that case, why not adding a real DOM element.